README.md

```diff
@@ -59,113 +53,144 @@ The following  NICs were tested with this implementation. However, other SRIOV c
 	- VF driver: v3.2.2-k
 > please refer to Intel download center for installing latest [Intel-® 82599ES 10 Gigabit Ethernet](https://ark.intel.com/products/41282/Intel-82599ES-10-Gigabit-Ethernet-Controller) drivers
 
-## Quick Start
-This section explains how to set up SRIOV Network device plugin in Kubernetes. Required YAML files can be found in [deployments/](deployments/) directory.
+### Environment
+This plugin was tested in following environment:
+- CentOS 7 - Kernel 4.13.7-1.el7.elrepo.x86_64
+- Go - version go1.10.3 linux/amd64
+- Kubernetes v1.12.0-alpha
```

Comment:
Is there special consideration on which kube release we will support or be compatible with going forward? 
for some feature like kubelet plugin watcher which is Beta in 1.12;  switch to it in DP might cause incompatibility with previous kube releases or some extra configuration(enable feature gates) in order to work with previous releases.

Reply:
It's depends on K8s device plugin features support really, our plugin should work with any k8s release that supports device plugin(without any dramatic changes of course). The Kubelet plugin watcher is backward compatible still. Later we should definitely switch to the plugin watcher mechanism. 
Here we simply mentioning the versions we have tested with this implementation.